lib/liquid/combinators/tags/generic.ex

```diff
@@ -7,5 +7,9 @@ defmodule Liquid.Combinators.Tags.Generic do
   """
   alias Liquid.Combinators.Tag
 
+  @type else_tag :: [
```

Comment:
Shouldn't an Else type be just a tagged tuple instead of a list of tagged tuples?

Reply:
Every tag is a keyword list (which is a list of tagged tuples)